src/plugins/telemetryTable/components/table.vue

```diff
@@ -667,6 +829,9 @@ export default {
         this.calculateScrollbarWidth();
 
         this.table.initialize();
+
+        document.addEventListener('keydown', this.keydown);
```

Comment:
Do we need these event listeners? Could we not just use `MouseEvent.ctrlKey` and `MouseEvent.metaKey`?

Reply:
I jumped the gun here a little, just need to make sure mouseEvent.ctrlKey and mouseEvent.metaKey works in a windows environment